client-evaluators/evaluatord/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "evaluatord"
```

Comment:
Nit: Can we move this entire tree to `sdk/client/engine` or similar? Also thinking about the naming I'm not sure if ending in 'd' makes sense as it's not really a dameon and more of a lib

Reply:
@markphelps You are right haha thank you it definitely is not a daemon 👌